scripts/build.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ev
 
+export BUILD_VERSION="0.0.2-SNAPSHOT"
```

Comment:
Are we planning on making this generated automatically? otherwise we need to find a better way to keep track of versioning inside the build script.